commercetools/commercetools-sdk-java-api/build.gradle

```diff
@@ -8,6 +8,7 @@ dependencies {
     implementation javax.validation
     api slf4j.api
     api commons.lang3
+    implementation 'org.jetbrains:annotations:23.0.0'
```

Comment:
do you know where this change came from?

Reply:
what I remember is that it was done automatically.
but I can remove it and see what happens

Reply:
I think it was wrongly committed because 1 week ago I was having issues with the annotations and the fact that the IDE didn't recognize them.
I'm reverting the changes.
Thanks for noticing it :D